libethash-cuda/CUDAMiner.h

```diff
@@ -106,7 +106,7 @@ class CUDAMiner: public Miner
 	void kick_miner() override;
 
 private:
-	bool m_abort = false;
+	atomic<bool> m_abort;
```

Comment:
You should initialize it here. Probably as `atomic<bool> m_abort = {false}`.

Reply:
Forgot I could do that! Ok.